samples/bluetooth/central_bas/README.rst

```diff
@@ -54,9 +54,9 @@ After programming the sample to your board, you can test it either by connecting
 Testing with another board
 --------------------------
 
-1. Connect the board that runs this sample to a terminal (for example, PuTTY).
-   See :ref:`putty` for the required settings.
-#. Program the other board with the :ref:`peripheral_hids_keyboard` or :ref:`peripheral_hids_mouse` sample.
+1. |connect_terminal|
+#. Reset the board.
+#. Program the other board with the :ref:`peripheral_hids_keyboard` or :ref:`peripheral_hids_mouse` sample and reset it.
```

Comment:
|connect_terminal| shortcut does not contain the text that this relates to the board programmed with this sample. Not sure if we can leave it as a conjectural (?) thing. - At least in places where we are using 2 boards.

Reply:
It should be clear from the context, but we can add a second shortcut to clarify. Never hurts to be more explicit. :)